analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DeclareTypesInNamespaces.AfterCSharp9.PartialProgramClass.cs

```diff
@@ -0,0 +1,4 @@
+﻿// reproducer for https://github.com/SonarSource/sonar-dotnet/issues/6836
+public partial class Program { } // Noncompliant, FP this is partial class
+                                 // for the class Program generated by the compiler when using top-level statements
```

Comment:
Can be on a single line
```suggestion
public partial class Program { } // Noncompliant, FP this is partial class for the class Program generated by the compiler when using top-level statements
```